models/pp_pop_psc_delta.h

```diff
@@ -387,12 +386,6 @@ pp_pop_psc_delta::set_status( const DictionaryDatum& d )
   State_ stmp = S_;      // temporary copy in case of errors
   stmp.set( d, ptmp );   // throws if BadProperty
 
-  // We now know that (ptmp, stmp) are consistent. We do not
-  // write them back to (P_, S_) before we are also sure that
-  // the properties to be set in the parent class are internally
-  // consistent.
-  Archiving_Node::set_status( d );
```

Comment:
@krishnanj This is actually intentional. The `pp_pop_psc_delta` model represents an entire population, not a single neuron, so STDP would not be well defined for it. Since spike archiving is only required for STDP, this model does not archive. See also the [comment by @mdeger](https://github.com/nest/nest-simulator/issues/77#issuecomment-159389434).